KatherineK/callcenter.py

```diff
@@ -0,0 +1,92 @@
+
+# "Hacker Level"
+
+#Create class CallCenter, with methods add, remove, removebyphone, info, and sort
+
+class CallCenter(object) :
+    def __init__(self) :
+        self.calls = []
+        self.queuesize = 0
+
+    def add(self, call) :
+        self.calls.append(call)
+        self.queuesize += 1
+
+    def remove(self, call) :
+        self.calls.remove(call)
+        self.queuesize -= 1
+
+    def removebyphone(self, phone) :
+        for calls in self.calls :
+            if calls.phone == phone :
+                self.remove(calls)
+        print "------------------"
+        self.info()
+
+    def info(self) :
+        for calls in self.calls :
+            calls.display()
+            print self.queuesize
+
+# Selection sorts by hour, then bubble sorts by min. Yes, I know I could just use .sort() but this is good practice
+    def sort(self) :
+        a = 0
+        temp = 0
+        for calls in self.calls :
+            for i in range(a, len(self.calls)) :
+                if self.calls[a].hour > self.calls[i].hour :
+                    temp = self.calls[a]
+                    self.calls[a] = self.calls[i]
+                    self.calls[i] = temp
+            a += 1
+        for calls in self.calls :
+            for i in range(a, len(self.calls)) :
+                if self.calls[a].hour == self.calls[i].hour and self.calls[a].min > self.calls[a].min :
+                    self.calls[a], self.calls[i] = self.calls[i], self.calls[a]
+        print "------------------"
+        self.info()
+
+
+# create class Call, with method display
+
+class Call(object):
+    id = 1
+    def __init__(self, name, phone, hour, min, reason, id=id):
+        self.id = Call.id
```

Comment:
This is a decent use case for having a global variable inside of a class. Good job!